cibyl/sources/zuul/transactions/__init__.py

```diff
@@ -394,16 +394,16 @@ def test(response):
         self._filters.append(test)
         return self
 
-    def with_status(self, *status: TestStatus) -> 'TestsRequest':
+    def with_status(self, *pattern: str) -> 'TestsRequest':
         """Will limit request to tests on a certain status.
 
-        :param status: Desired status of the test cases.
+        :param pattern: Regex pattern for the desired test status.
         :return: The request's instance.
         """
 
         def test(response):
             return any(
-                response.status == patt for patt in status
+                matches_regex(patt, str(response.status)) for patt in pattern
```

Comment:
Both Jenkins and Elasticsearch support case-insensitive filtering by result. It would be good to have also in zuul, and I think it should be a quick change. To clarify, I mean matching both 'SUCCESS' and 'success'.

Reply:
How do you propose doing that? I could either cast the string to lowercase or use the 're.IGNORECASE' flag on the regex matcher.

Reply:
Both options seem good to me, go for the one that is simpler for you, maybe using the IGNORECASE flag is simpler in this case.

Reply:
Done, take a look when you can.